PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * ONNXRuntime.h
+ *
+ *  Created on: Jun 28, 2019
+ *      Author: hqu
+ */
+
+#ifndef PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+#define PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+
+#include <vector>
+#include <map>
+#include <string>
+#include <memory>
+
+#include "onnxruntime/core/session/onnxruntime_cxx_api.h"
+
+namespace Ort {
+
+  typedef std::vector<std::vector<float>> FloatArrays;
+
+  class ONNXRuntime {
+  public:
+    ONNXRuntime();
+    ONNXRuntime(const ONNXRuntime&) = delete;
+    ONNXRuntime& operator=(const ONNXRuntime&) = delete;
+    virtual ~ONNXRuntime();
+
+    // Creates session
+    void createSession(const std::string& model_path, const SessionOptions* session_options = nullptr);
```

Comment:
Can the functionality of `createSession()` be moved to the constructor? By quick look all uses in this PR are of the form "construct+call `createSession()`.

Reply:
Good point!